src/go/rpk/pkg/tuners/network/nics.go

```diff
@@ -107,6 +107,10 @@ func GetRpsCPUMask(
 		return "0x0", nil
 	}
 
+	if !t.GetAllowRpsRfsTuner() {
```

Comment:
Does it make sense to move it up, above many/all of the error conditions? E.g., if you disable RFS/RPS maybe you don't want to get errors related to calculating the RFS/RPS mask which you wouldn't use anyway?